cordapp/README.md

```diff
@@ -1,13 +1,24 @@
 # CorDapp Sample - Java
```

Comment:
I would rename Java from the Headline.
We can mention this in the first sentence - `This is a simple CorDapp written in Java using the Conclave API.` instead.

Reply:
Done.